integrationtests/tests/tests.rs

```diff
@@ -374,7 +376,7 @@ async fn drop_peers_who_dont_contribute_decryption_shares() -> Result<()> {
     let invoice = tokio::join!(
         user.client
             .generate_invoice(payment_amount, "".into(), rng(), None),
-        fed.await_consensus_epochs(1) // create offer
+        fed.await_consensus_epochs(2) // create offer
```

Comment:
The number of epochs were increased due to tests timing out. It seems like adding the consensus item for clock time requires adding a consensus round, but this is my first time working on consensus code so not 100% sure.

Reply:
Need to look into this in more detail, maybe @jkitman has some intuition.

Reply:
It looks like after adding `LightningConsensusItem`, the LN module will continually produce epochs all the time.

However, we want to avoid this behavior to reduce the amount of unnecessary bandwidth/storage/compute that nodes use.

I suggest modifying `await_consensus_proposal` in the LN module to continue only when the time will generate a change in invoice expiry.

Reply:
Added logic in [`consensus_proposal`](https://github.com/fedimint/fedimint/pull/1074/commits/1ea20954bb029c09c36cf2febdcf621f064a5bac#diff-d777522b5e9ff491a0359a5506622730cabc67755d33cfab1a5ebf3cf1b0738dR309-R337) to only propose new clock time if an invoice would expire to avoid continually producing epochs